### ISSUE TITLE
Changing/saving setting for BYML Inline Container Max Count crashes program

### ISSUE THREAD

**Issue by PerlDude81** [EXKing-Editor/EXKing-Editor#72]
With a test-case of a single user (me) I can't be sure it's not something I'm doing wrong.

Trying to change the value for the BYML Inline Container Max Count crashes the program for me when I click save. I'm on version 1.4.0.

Using Windows video capture on the program window (last frame is when NxEditor crashed):

https://github.com/user-attachments/assets/11d63b89-0eff-4b58-a592-8c7c89260ef9

If it's not a bug - and something I'm doing wrong - please close/ignore. Thank you.

Windows 10 Pro
Version: 22H2
Build: 19045.4780
Windows Feature Experience Pack 1000.19060.1000.0

**Comment by ArchLeaders** [comment]
Please send the latest log file in `%appdata%\nx-editor\logs` (`%appdata%\nx-editor` may be different if you've changed the path in the settings)

I've never had this issue before, so I'm unsure what it could be.

Do you know if it only occurs when saving that setting, or just saving the settings in general?

**Comment by PerlDude81** [comment]
You know what, now that you mention it, it is crashing any time I click the "Save" button. The only thing I changed before is the Storage Folder setting, which is on a different drive than my OS - if that matters. It saved that setting a long time ago.

Changing a setting, changes them in the running instance but does not save the changes.

Github didn't like the .yml filetypes, so pasting contents of last four log files:

```
Loaded NxEditor.EpdPlugin.dll
16:20:00:014 [LoadPlugin]: Loaded NxEditor.EpdPlugin.dll
Loaded NxEditor.TotkPlugin.dll
16:20:00:019 [LoadPlugin]: Loaded NxEditor.TotkPlugin.dll

Loaded NxEditor.EpdPlugin.dll
16:21:01:046 [LoadPlugin]: Loaded NxEditor.EpdPlugin.dll
Loaded NxEditor.TotkPlugin.dll
16:21:01:050 [LoadPlugin]: Loaded NxEditor.TotkPlugin.dll
Loaded NxEditor.EpdPlugin.dll
16:21:24:658 [LoadPlugin]: Loaded NxEditor.EpdPlugin.dll
Loaded NxEditor.TotkPlugin.dll
16:21:24:662 [LoadPlugin]: Loaded NxEditor.TotkPlugin.dll

Loaded NxEditor.EpdPlugin.dll
16:22:11:785 [LoadPlugin]: Loaded NxEditor.EpdPlugin.dll
Loaded NxEditor.TotkPlugin.dll
16:22:11:789 [LoadPlugin]: Loaded NxEditor.TotkPlugin.dll
16:24:32:119 [LoadEditor]: Processing Weapon_Spear_030.pack.zs
16:24:34:780 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Base.engine__actor__ActorParam.bgyml
16:24:37:798 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_030.engine__actor__ActorParam.bgyml
16:24:40:375 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_173.engine__actor__ActorParam.bgyml
16:24:42:785 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_Base.engine__actor__ActorParam.bgyml
16:24:45:101 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_Metal_Base.engine__actor__ActorParam.bgyml
16:24:51:631 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Equip_UseTake.game__attention__ClientParam.bgyml
16:24:59:502 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_030.game__component__AttachmentParam.bgyml
16:25:02:677 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_Base.game__component__AttachmentParam.bgyml
16:25:06:351 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_030.engine__component__ModelInfo.bgyml
16:25:10:126 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_001_Metal.engine__component__PhysicsParam.bgyml
16:25:18:340 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/WeaponDefault.game__component__PouchContentParam.bgyml
16:25:24:075 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Direct_Spear_Metal.engine__component__SLinkParam.bgyml
16:25:28:912 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/WeaponCommon.game__component__SpecialPowerReceiverParam.bgyml
16:25:45:026 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_030.engine__component__ModelInfo.bgyml
16:25:49:585 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_001_Metal.engine__component__PhysicsParam.bgyml

Loaded NxEditor.EpdPlugin.dll
16:26:05:854 [LoadPlugin]: Loaded NxEditor.EpdPlugin.dll
Loaded NxEditor.TotkPlugin.dll
16:26:05:899 [LoadPlugin]: Loaded NxEditor.TotkPlugin.dll
16:26:11:200 [LoadEditor]: Processing Weapon_Spear_030.pack.zs
16:26:18:562 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/Weapon_Spear_Base.game__component__WeaponParam.bgyml
16:26:25:138 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/EquipmentCommon.game__life__DamageParameters.bgyml
16:26:30:102 [LoadEditor]: Processing Weapon_Spear_030.pack.zs/WeaponCommon.game__component__SpecialPowerReceiverParam.bgyml

```

**Comment by ArchLeaders** [comment]
Looks like my logging is still horrible.

Try running NXE through CMD and see if the error shows up there.

If the console window disappears, use `View > Show/Hide Console` to restore it.

(Open CMD in Windows Search, type in `nxe`, press enter and use normally, then you can send a screenshot of the CMD window or copy/paste the output.)

**Comment by PerlDude81** [comment]
Okay, I did that...

The output was dying first on saying recent.json already existed. Then it died saying a log file from back in February already existed. Like it was looking in the default storage folder.

Turns out, the error exists between my keyboard and chair...

![image](https://github.com/user-attachments/assets/6efc63ba-ee66-4aef-b778-2c75770b27e0)

I never did that. I'm guessing it means the old folder must be deleted if you're changing the storage location?

Anyway, thanks for the help and sorry for making a fuss out of something that was there - literally in black and white - in your instructions. I tried a bunch of things first before reaching out.
